### ISSUE TITLE
Color background on search bar

### ISSUE THREAD

**Issue by Toinane** [electron/electronjs.org#5299]
I don't think this issue was already reported. When you go to see the page of an application, the background's color of the header can change to the color of the application in question, but the background's color of the search bar didn't change and keep the default color.

<img width="484" alt="Screenshot 2021-04-07 at 16 15 55" src="https://user-images.githubusercontent.com/7584391/113881885-1d3bbf00-97bd-11eb-87b9-088f4126066c.png">

And depending on the color, it can be quite ugly 😄 
Also, the text color is black, it's not very readable and not accessible

<img width="247" alt="Screenshot 2021-04-07 at 16 22 45" src="https://user-images.githubusercontent.com/7584391/113882426-963b1680-97bd-11eb-9f1a-2ae501547682.png">

**Comment by JamMor** [comment]
It seems like the app colors are passed in outside the available code here. If they could pass a variable with half of the saturation of goodColoronWhite (the app header color), it would make a good search background color for that app page.

There are less clean solutions to modify the color in here if that isn't an option though

**Comment by Gruce** [comment]
It would be prettier if they put it like `#0d0d0d94`

**Comment by abhijain2003** [comment]
I want to work on this issue can I do it
